### PR TITLE
Node examples

### PR DESCRIPTION
**Pull request by ryannguyen16**
Creating examples for Node library

### CODE REVIEW

File: official/docs/node/addresses/create_addresss.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    street2: 'FLOOR 5',
+    city: 'SAN FRANCISCO',
+    state: 'CA',
+    zip: '94104',
+    country: 'US',
+    company: 'EasyPost',
+    phone: '415-123-4567',
+  });
+  
+  console.log(address);
```
Comment (Justintime50) [inline official/docs/node/addresses/create_addresss.js:15]: For all of these examples, you'll need to use `then` to chain together the logging to console (see our current docs for an example), this is required based on how the Node library is setup as we use async/await here. Basically "do this thing, then when it's done do the next thing", if we try console logging without the `then`, the API call may not be done and there won't be anything to log yet. This is true of every example.

```suggestion
const address = new api.Address({
    street1: '417 MONTGOMERY ST',
    street2: 'FLOOR 5',
    city: 'SAN FRANCISCO',
    state: 'CA',
    zip: '94104',
    country: 'US',
    company: 'EasyPost',
    phone: '415-123-4567',
  }).then(console.log);
```

**Review by Justintime50 — DISMISSED** [review]
I added the linter and formatter. There should be a dozen or so linting errors that need fixing (run `make lint-node` to see the output) and there are a few formatting issues that need fixing (most should be able to be fixed by running `npm run format`), let me know if you have questions on these!

File: official/docs/node/addresses/create_addresss.js (hunk 1 above)
Comment (jchen293) [inline official/docs/node/addresses/create_addresss.js:15]:
```suggestion
console.log(address);
```
extra spaces, please check this on other files as well

File: official/docs/node/addresses/create_addresss.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    street1: '417 MONTGOMERY ST',
+    street2: 'FLOOR 5',
+    city: 'SAN FRANCISCO',
+    state: 'CA',
+    zip: '94104',
+    country: 'US',
+    company: 'EasyPost',
+    phone: '415-123-4567',
+  });
+
```
Comment (jchen293) [inline official/docs/node/addresses/create_addresss.js:14]: You need to use `save` function to actually create the address

address.save().then(console.log);

Please check on other function that is creating an object, make sure they are using `save` function

File: official/docs/node/addresses/create_and_verify_address.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+const address = new api.Address.createAndVerify({
```
Comment (jchen293) [inline official/docs/node/addresses/create_and_verify_address.js:4]: Most of the functions in Node.js library are async, you have to use [await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) keyword or [then](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then)

This applies to some other code examples, you can look at API doc or unit tests for reference

File: official/docs/node/batches/manifest_batch.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+api.Batch.retrieve('batch_...').then(b => {
+    b.createScanForm().then(console.log);
+  });
+
```
Comment (jchen293) [inline official/docs/node/batches/manifest_batch.js:7]: You need a new line at the end of file

File: official/docs/node/shipments/convert_label.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+api.Shipment.retrieve('shp_...').then(shipment => {
+    shipment.convertLabelFormat('ZPL').then(console.log);
+  });
+
```
Comment (jchen293) [inline official/docs/node/shipments/convert_label.js:7]: You need a new line at the end of file

**Review by jchen293 — DISMISSED** [review]

**Comment by ryannguyen16** [comment]
fixed formatting errors today and will fix the rest tomorrow

**Review by nwithan8 — APPROVED** [review]

File: official/docs/node/pickups/buy_pickup.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const Easypost = require('@easypost/api');
+const api = new Easypost('EASYPOST_API_KEY');
+
+api.Pickup.retrieve('pickup_...').then((p) => {
+  p.buy('UPS', 'Same-day Pickup').then(console.log);
+});
```
Comment (jchen293) [inline official/docs/node/pickups/buy_pickup.js:6]:
```suggestion
api.Pickup.retrieve('pickup_...').then((retrievePickup) => {
  retrievePickup.buy('UPS', 'Same-day Pickup').then(console.log);
});
```

These variable names could be more explicit, this also applied to other files that are using a single character as the variable names.

**Review by jchen293 — APPROVED** [review]

**Pull request merged**